apps/publish/content/resend.py

```diff
@@ -72,6 +72,12 @@ def _validate_subscribers(self, subscriber_ids, article):
     def _validate_article(self, article_id, article_version):
         archive_article = get_resource_service(ARCHIVE).find_one(req=None, _id=article_id)
 
+        if app.config.get("CORRECTIONS_WORKFLOW") and archive_article.get(ITEM_STATE) == "correction":
+            publish_service = get_resource_service("published")
+            archive_article = publish_service.find_one(
```

Comment:
this is not `archive_article` really, not sure why it wouldn't find the article by `article_id` ?

Reply:
When the correction workflow config is set and we try to correct the item, it changes the state of the item present inside archive collection as `correction` and the item present inside the published collection as `being_corrected` whereas if we publish an item, it will change the state to `published` in both the collections (archive collection and published collection)

if I use this: `archive_article = get_resource_service(ARCHIVE).find_one(req=None, _id=article_id)`, it will return the item which is present inside the archive collection and the item present there has a state of `correction` but we need the item which has a state of `being_corrected` which is present inside the published collection.